www/js/model.js

```diff
@@ -199,6 +199,11 @@ define([
 				var node = this.getNode(id);
 				return node && node.expression;
 			},
+			getInputs: function(/*string*/ id){
+				// Summary: return an array containing the input ids for a node.
+				var ret = this.getNode(id);
+				return ret && ret.inputs;
```

Comment:
inputs were named link in the data object. Links getter and setter are already part of the model